src/components/MultiHopTrade/components/RateGasRow.tsx

```diff
@@ -29,13 +30,12 @@ import { Amount } from '@/components/Amount/Amount'
 import { HelperTooltip } from '@/components/HelperTooltip/HelperTooltip'
 import { Row } from '@/components/Row/Row'
 import { RawText, Text } from '@/components/Text'
-import { selectVotingPower } from '@/state/apis/snapshot/selectors'
 import { selectAssetById } from '@/state/slices/selectors'
 import { useAppSelector } from '@/state/store'
 import { clickableLinkSx } from '@/theme/styles'
 
 type RateGasRowProps = {
-  affiliateBps: string | undefined
+  affiliateBps: BigNumber.Value
```

Comment:
The no undefined mentioned above - assumes a value is always present (whatever that value is, which should always be 55)